scripts/inkeep-local.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
@miles-kt-inkeep did you test this functionality. It might be residual from the first commits.